build.gradle

```diff
@@ -13,6 +13,8 @@ buildscript {
             name = 'forge'
             url = 'http://files.minecraftforge.net/maven'
         }
+        maven { url 'https://jitpack.io' }
+        mavenLocal()
```

Comment:
This line exists now for debugging purposes, right? Should remove it before official merge.